### PR TITLE
Add notebook for basic metpy calculations

### LINKED ISSUE #30: Create a "MetPy (basic calculations)" Training Notebook
Following the [instructors/notebook-template/Template.ipynb](https://github.com/Unidata/pyaos-ams-2021/blob/master/instructors/notebook-template/Template.ipynb) template notebook, create a training notebook that lives in `notebooks/analysis/` directory that covers the basics of using MetPy to perform meteorological computations to assist workshop participants who choose to work on the following projects:
  * [`site/projects/isentropic_analysis.md`](https://unidata.github.io/pyaos-ams-2021/projects/isentropic_analysis.html)
  * [`site/projects/severe_wx_outbreak.md`](https://unidata.github.io/pyaos-ams-2021/projects/severe_wx_outbreak.html)

Once the notebook is complete, be sure to do the following:

* [ ] clear the output of the notebook
  * jupyter-notebook interface: `Cell` &rarr; `All Output` &rarr; `Clear`
  * jupyter-lab interface: `Right-click` &rarr; `Clear All Outputs`
* [ ] link the notebook in the following locations using a nbviewer url (`https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/<training-notebook-name>.ipynb`):
  * [ ] the project page for each project idea listed above
  * [ ] the project entry in [`site/resources.md`](https://unidata.github.io/pyaos-ams-2021/resources.html)
* [ ] add a description of the notebook to project entry in [`site/resources.md`](https://unidata.github.io/pyaos-ams-2021/resources.html)

### PR DESCRIPTION
**Pull request by tjwixtrom**
Provides some basic `metpy.calc` usage examples with fake data. Closes #30.

### CODE REVIEW

File: site/resources.md [hunk 1]
```diff
@@ -73,9 +73,9 @@ Resources
 
   _add description_
 
-* MetPy (basic calculations)
+* [MetPy (basic calculations)](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/metpy_basics.ipynb)
```
Comment (srcarter3) [inline site/resources.md:76]: Please bold the title, remove the new line between the title and description, and add two blank spaces to the end of the title, after the link.  (see the python-awips entries in this markdown for reference if needed)

File: site/resources.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ Resources
 
 * [**python-awips: Working with the Maps and Topography Databases**](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/visualization/python-awips-WorkingWithMapsTopoDatabases.ipynb)  
   This notebook covers how to use the AWIPS Maps Database to access GIS objects, which are returned as Shapely geometries (Polygon, Point, MultiLineString, etc.) and can be easily plotted by Matplotlib, Cartopy, MetPy, and other packages.  It shows how to use maps and topo data types to obtain GIS data from the AWIPS databases.  Finally, it walks through the steps of plotting data to create a very informative map of a County Warning Area.
-  
+
```
Comment (srcarter3) [inline site/resources.md:126]: Not sure this is needed but I don't think it affects the rendering of the page.

**Review by srcarter3 — CHANGES_REQUESTED** [review]
Looks good!  One minor markdown change for the resources page.

Also, don't forget to add nbviewer links to the "Severe Weather Outbreak Case Study" (severe_wx_outbreak.md), and "Isentropic Analysis Case Study" (isentropic_analysis.md) at some point (doesn't have to be this PR).  Thanks!

File: site/resources.md [hunk 3]
```diff
@@ -70,9 +70,9 @@ Resources
 
   _add description_
 
-* MetPy (basic calculations)
+* [**MetPy (basic calculations)**](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/metpy_basics.ipynb)
```
Comment (srcarter3) [inline site/resources.md:74]: Thanks for bolding the title.
Now for the proper spacing between the title and description, remove the new line, and add two spaces at the end of the title link.  If you pull from the master version first, you'll notice a new comment at the top of this file, explaining that the spaces are necessary for markdown to interpret and render the webpage properly.

**Review by srcarter3 — CHANGES_REQUESTED** [review]
Markdown is picky and needs 2 spaces and no new line to render the description properly under the title.

File: site/resources.md (hunk 3 above)
Comment (tjwixtrom) [reply site/resources.md]: Ok, try that.

**Review by srcarter3 — APPROVED** [review]

File: site/resources.md (hunk 3 above)
Comment (srcarter3) [reply site/resources.md]: Great!  Thanks.

**Pull request merged**